deploy/gcp/manifests/crd.yaml

```diff
@@ -1 +1 @@
-../../../manifests/crd.yaml
```

Comment:
What's the change here?

Reply:
The following files should be removed:
manifests/crd/v1/pingcap.com_tidbclusters.yaml
manifests/crd/v1beta1/pingcap.com_tidbclusters.yaml

Reply:
done